projects/ion/src/lib/utils/baseTable.ts

```diff
@@ -74,16 +77,26 @@ export abstract class BaseTable<
     }
 
     return upArrow
-      ? this.fillColorArrowUp(column)
-      : this.fillColorArrowDown(column);
-  }
-
-  public fillColorArrowUp(column: Column): string {
-    return column.desc ? DISABLED_COLOR : ENABLED_COLOR;
-  }
-
-  public fillColorArrowDown(column: Column): string {
-    return column.desc ? ENABLED_COLOR : DISABLED_COLOR;
+      ? this.fillArrow(column, 'up')
+      : this.fillArrow(column, 'down');
+  }
+
+  public fillArrow(column: Column, direction: string): string {
+    const theme = JSON.parse(localStorage.getItem('ion-theme'));
+    const isDarkTheme = theme.key === 'dark';
```

Comment:
There is a better way to get the current theme: using the "IonThemeService", because if in the future any changes are made to the way the theme is stored, this component will continue ok.

after injecting the service into this class, you can use something like this:
```suggestion
    const isDarkTheme = this.ionThemeService.theme.key === 'dark';
```